csharp/test/Microsoft.ML.OnnxRuntime.Tests.NetCoreApp/InferenceTest.netcore.cs

```diff
@@ -351,7 +351,16 @@ private void TestTensorRTProviderOptions()
                 // the expansion of Softplus uses Exp(1). ORT has a Softplus kernel, so testing the expansion is
                 // unnecessary and fails as ORT support for Exp started at opset 6 (as ORT didn't exist until opset 7).
                 { "test_softplus_example_expanded", "Not applicable"},
-                { "test_softplus_expanded", "Not applicable"}
+                { "test_softplus_expanded", "Not applicable"},
+
+                // opset 18 models. still failed after ORT 1.14 released
+                { "test_bitwise_or_i16_4d", "pending opset 18 support"},
+                { "test_col2im_pads", "pending opset 18 support"},
```

Comment:
Can we add a note this is due to a typo currently? https://github.com/microsoft/onnxruntime/pull/12311#issuecomment-1398973366